storage/plugins/book-club/src/ChallengeController.php

```
@@ -0,0 +1,202 @@
+            return $this->notFound($response);
+        }
+        $clubId = (int) $club['id'];
+        $userId = $this->userId();
+        $year = (int) date('Y');
+
+        // Lazy recompute so the page never shows stale numbers, even when
+        // the maintenance cron has not run yet today.
+        try {
+            $this->challenges->recomputeClub($clubId, $year);
+        } catch (\Throwable $e) {
+            SecureLogger::warning('[BookClub:challenges] lazy recompute failed for club ' . $clubId . ': ' . $e->getMessage());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\SecureLogger' in method 'show'. (undefined)

(StaticAccess)